eng/native/configurecompiler.cmake

```diff
@@ -193,6 +193,10 @@ if(CLR_CMAKE_HOST_LINUX)
   add_compile_options($<$<COMPILE_LANGUAGE:ASM>:-Wa,--noexecstack>)
   add_linker_flag(-Wl,--build-id=sha1)
   add_linker_flag(-Wl,-z,relro,-z,now)
+  if(CLR_CMAKE_HOST_ALPINE_LINUX)
+    add_linker_flag(-lucontext)
+	add_linker_flag(-Wl,-z,notext)
```

Comment:
can you please add a comment explaining why we need this on alpine?

Reply:
Absolutely.